### ISSUE TITLE
Dragon's lair starts with Dirk having already successfully defeated the tentacle monster below the draw bridge

### ISSUE THREAD

**Issue by triggerthehorizon** [icculus/DirkSimple#27]
Dragon's lair starts with Dirk having already successfully defeated the tentacle monster below the draw bridge

**Comment by icculus** [comment]
This is how the US arcade version works. My understanding is that the arcade cabinets in Europe added this sequence back in (as did the Dragon's Lair Enhancement ROM).

Eventually I plan to put an option in to enable this sequence, but I haven't yet.

**Comment by triggerthehorizon** [comment]
I just checked arcade gameplay footage and you are right. I played so many versions of this on 3do sega cd, pc, ios, I was sure that bridge tentacle monster thing was part of the arcade game as well! It is kind of weird right now because the “success” music plays as he runs through the gate when the game starts.

**Comment by icculus** [comment]
Yeah, the original version was trying to rush players into the main game as fast as possible, and just about every rerelease of the game on other platforms added this back in.

We will be adding it, as an option, too.
